qutip/solver/mesolve.py

```diff
@@ -20,110 +20,110 @@ def mesolve(H, rho0, tlist, c_ops=None, e_ops=None, args=None, options=None,
-          How to present the solver progress.
-          'tqdm' uses the python module of the same name and raise an error
-          if not installed. Empty string or False will disable the bar.
-        - progress_kwargs : dict
-          kwargs to pass to the progress_bar. Qutip's bars use `chunk_size`.
-        - method : str ["adams", "bdf", "lsoda", "dop853", "vern9", etc.]
-          Which differential equation integration method to use.
-        - atol, rtol : float
-          Absolute and relative tolerance of the ODE integrator.
-        - nsteps :
-          Maximum number of (internally defined) steps allowed in one ``tlist``
-          step.
-        - max_step : float, 0
-          Maximum lenght of one internal step. When using pulses, it should be
-          less than half the width of the thinnest pulse.
+        - | store_final_state : bool
+          | Whether or not to store the final state of the evolution in the
+            result class.
+        - | store_states : bool, None
+          | Whether or not to store the state vectors or density matrices.
+            On `None` the states will be saved if no expectation operators are
+            given.
+        - | normalize_output : bool
+          | Normalize output state to hide ODE numerical errors.
+        - | progress_bar : str {'text', 'enhanced', 'tqdm', ''}
+          | How to present the solver progress.
+            'tqdm' uses the python module of the same name and raise an error
+            if not installed. Empty string or False will disable the bar.
+        - | progress_kwargs : dict
+          | kwargs to pass to the progress_bar. Qutip's bars use `chunk_size`.
+        - | method : str ["adams", "bdf", "lsoda", "dop853", "vern9", etc.]
+          | Which differential equation integration method to use.
+        - | atol, rtol : float
+          | Absolute and relative tolerance of the ODE integrator.
+        - | nsteps : int
+          | Maximum number of (internally defined) steps allowed in one ``tlist``
+            step.
+        - | max_step : float
+          | Maximum lenght of one internal step. When using pulses, it should be
+            less than half the width of the thinnest pulse.
 
         Other options could be supported depending on the integration method,
         see `Integrator <./classes.html#classes-ode>`_.
 
     Returns
     -------
-    result: :class:`qutip.Result`
+    result: :obj:`.Result`
 
-        An instance of the class :class:`qutip.Result`, which contains
-        a *list of array* `result.expect` of expectation values for the times
-        specified by `tlist`, and/or a *list* `result.states` of state vectors
-        or density matrices corresponding to the times in `tlist` [if `e_ops`
-        is an empty list of `store_states=True` in options].
+        An instance of the class :obj:`.Result`, which contains a *list of
+        array* ``result.expect`` of expectation values for the times specified
```

Comment:
Can inline markdown be split across lines like this? I.e. does `* list of array*` still render correctly?

Reply:
It does when I build it locally.